src/Concordium/Client/Output.hs

```diff
@@ -787,27 +793,48 @@ showEvent verbose stM = \case
     showDelegationTarget Types.DelegatePassive = "Passive delegation"
     showDelegationTarget (Types.DelegateToBaker bid) = "Baker ID " ++ show bid
 
+    -- Attempt to decode a bytestring according to a schema.
+    -- If there is no schema, or decoding fails @Nothing@ is returned.
```

Comment:
It's just a helper for `eventToString` and `parameterToString` which conveniently takes a schema type wrapped in Maybe. Since we do not intend to use the error message, the semantics are that **in case of a decoding failure** or **if a schema is not present**, Nothing is returned.

Should I modify it to return the error? In that case, then I believe the signature should probably be:

`toJSON' :: CS.SchemaType -> BSS.ShortByteString -> Either String String`

or

`toJSON' :: Maybe CS.SchemaType -> BSS.ShortByteString -> Either String String`

I am not sure whether this adds any benefits, or makes anything simpler?

Reply:
Ok.If it is going to be ignored then we don't have to keep it. But maybe add what you wrote above (about the error not being useful) to the documentation.